js/src/shell/index.js

```diff
@@ -85,13 +85,12 @@ function onEnterDapp ({ params }) {
 
 ReactDOM.render(
   <ContextProvider api={ api } muiTheme={ muiTheme } store={ store }>
-    <Router history={ hashHistory }>
-      <Route path='/' component={ Application }>
-        <Redirect from='/auth' to='/' />
+    <Router>
+      <Application>
+        <Route path='/' exact component={ Dapps } />
```

Comment:
Yep. Havn't solved that one yet -_- There is another issue with clicking an account [accounts -> account] It pushes a '#/' prior to the account. 

I really think it would be better to have accounts and account all in one dapp if that is possible.